tests/data_context/test_get_data_context.py

```diff
@@ -148,11 +155,23 @@ def test_cloud_missing_env_throws_exception(clear_env_vars, empty_ge_cloud_data_
 
 @pytest.mark.parametrize("params", [GX_CLOUD_PARAMS_REQUIRED, GX_CLOUD_PARAMS_ALL])
 @pytest.mark.cloud
-def test_cloud_context_params(monkeypatch, empty_ge_cloud_data_context_config, params):
-    with mock.patch.object(
-        CloudDataContext,
-        "retrieve_data_context_config_from_cloud",
-        return_value=empty_ge_cloud_data_context_config,
+def test_cloud_context_params(
+    unset_gx_env_variables, monkeypatch, empty_ge_cloud_data_context_config, params
```

Comment:
I've added unset_gx_env_variables to a few tests that don't need env vars to make them behave the same locally and in ci.